hack/gatewayapi-conformance.sh

```diff
@@ -47,7 +47,7 @@ go mod vendor
 # because the AWS ELB needs extra ~60s for DNS propagation
 sed -i "s/MaxTimeToConsistency:              30/MaxTimeToConsistency:              90/g" conformance/utils/config/timeout.go
 
-SUPPORTED_FEATURES="Gateway,HTTPRoute,ReferenceGrant,GatewayPort8080,HTTPRouteQueryParamMatching,HTTPRouteMethodMatching,HTTPRouteResponseHeaderModification,HTTPRoutePortRedirect,HTTPRouteSchemeRedirect,HTTPRoutePathRedirect,HTTPRouteHostRewrite,HTTPRoutePathRewrite,HTTPRouteRequestMirror,HTTPRouteRequestMultipleMirrors,HTTPRouteBackendProtocolH2C,HTTPRouteBackendProtocolWebSocket"
+SUPPORTED_FEATURES='Gateway,HTTPRoute,ReferenceGrant,GatewayHTTPListenerIsolation,GatewayInfrastructurePropagation,GatewayPort8080,GatewayStaticAddresses,HTTPRouteBackendProtocolH2C,HTTPRouteBackendProtocolWebSocket,HTTPRouteBackendRequestHeaderModification,HTTPRouteBackendTimeout,HTTPRouteDestinationPortMatching,HTTPRouteHostRewrite,HTTPRouteMethodMatching,HTTPRouteParentRefPort,HTTPRoutePathRedirect,HTTPRoutePathRewrite,HTTPRoutePortRedirect,HTTPRouteQueryParamMatching,HTTPRouteRequestMirror,HTTPRouteRequestMultipleMirrors,HTTPRouteRequestTimeout,HTTPRouteResponseHeaderModification,HTTPRouteSchemeRedirect,MeshConsumerRoute'
```

Comment:
Some of these features look like extended features, which we don't need to test. It would save some time not to test them.  We should only need to test the standard features, excluding mesh.  That would be any feature at the code links below that is listed in the `FeatureChannelStandard`. 

Gateway: https://github.com/kubernetes-sigs/gateway-api/blob/main/pkg/features/gateway.go#L65-L86

HTTPRoute: https://github.com/kubernetes-sigs/gateway-api/blob/main/pkg/features/httproute.go#L103-L193 

GRPCRoute: https://github.com/kubernetes-sigs/gateway-api/blob/main/pkg/features/grpcroute.go#L31-L34

ReferenceGrant: https://github.com/kubernetes-sigs/gateway-api/blob/main/pkg/features/referencegrant.go#L31-L34

Reply:
Thanks!  Fixed in https://github.com/openshift/cluster-ingress-operator/compare/30b576c4293b2962b137bcb93ff77f278d7bf681..26a3918c7374cc16eac04b87c9e4267591c304a5.  I still get some failures, though, and the test takes over an hour to run (it didn't finish before my cluster expired).